interface/main/finder/dynamic_finder.php

```diff
@@ -104,7 +102,45 @@
  });
 
  // OnClick handler for the rows
- $('#pt_table tbody tr').live('click', function () {
+
+  $(document).on('keyup', '#textID', function (e) {
+	 var code = e.which;
+
+	 if(code == 13){
+		 
+		  // ID of a row element is pid_{value}
+		  var newpid = $('#pt_table tbody tr').attr('id').substr(4);
+		  // If the pid is invalid, then don't attempt to set 
```

Comment:
It looks like the comments are related to the javascript, I suggest leaving them as it. Wrapping comments in PHP just to hide them from the browser is not something we want to get in the habit of doing.

There are ways of removing comments from JS, but it's not something I suggest right now; it'd be a very large undertaking.